README.md

```diff
@@ -112,8 +112,7 @@ kind: QuayEcosystem
 metadata:
   name: example-quayecosystem
 spec:
-  quay:
-    imagePullSecretName: redhat-pull-secret
+  quay: {}
```

Comment:
What do you think about adding a yaml comment at the above all of the `redhat-pull-secret` lines that says, "Required only if 'quay' and 'clair-jwt' images require pull secrets"

Reply:
alternatively, could put two example blocks, one with and one without the imagePullSecretName

Reply:
im hesitant as that would really clutter up the documentation.

Reply:
How about two example blocks just here, one with and one without the secret. The rest merely omit.